src/System.Security.AccessControl/src/System/Security/AccessControl/PrivilegeNotHeldException.cs

```diff
@@ -28,6 +30,22 @@ public PrivilegeNotHeldException(string privilege, Exception inner)
             _privilegeName = privilege;
         }
 
+        internal PrivilegeNotHeldException(SerializationInfo info, StreamingContext context) : base(info, context)
+        {
+            _privilegeName = info.GetString(nameof(PrivilegeName));
+        }
+
+        public override void GetObjectData(SerializationInfo info, StreamingContext context)
+        {
+            if (info == null)
+            {
+                throw new ArgumentNullException(nameof(info));
+            }
+
+            base.GetObjectData(info, context);
+            info.AddValue(nameof(PrivilegeName), _privilegeName, typeof(string));
```

Comment:
We should really have serialization tests for everything that's marked [Serializable] (since even without a custom serialization implementation, serialization could fail if it has a member that can't be serialized).  I plan to sweep through all of them adding tests.  Right now I'm trying to get all of the implementations there, with tests for the ones that have the more complicated implementations.